### DISCUSSION TITLE
Password hash

### DISCUSSION THREAD

**Discussion by Fbuona** [directus/directus#21390] [Feature Requests]
Hi, 
It is possible ti setup Directus not to hash the user's password ? 
I'm already encrypting my users password in my app and I just want Directus to store the md5 encrypted password without hashing it in argon2id.

Would it be possible ? 

Thanks.

Fabio

**Comment by ched-dev** [comment]
It is not possible to do that now and I doubt it would be added. MD5 is a weak password hashing algorithm due to the speed of computers these days.

**Comment by br41nslug** [comment]
If you're already hashing the password then you shouldn't be using the built-in Directus Users table but rather a custom users table to store the accounts for your external system. As ched-dev mentions it is highly recommended to no longer use MD5 for any security critical things as it has been cracked quite a while ago and will not be implemented on the Directus side of things.

**Comment by Fbuona** [comment]
Ok thanks for your help, I'll take your feedbacks into account!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
